tests/devices/test_default_qubit_autograd.py

```diff
@@ -368,3 +368,21 @@ def cost(weights):
 
         grad = qml.grad(cost)(weights)[0]
         assert grad.shape == weights.shape
+
+class TestOps:
+    """Unit tests for operations supported by the default.qubit.autograd device"""
+
+    def test_multirz_jacobian(self):
+        """Test that the patched numpy functions are used for the MultiRZ
+        operation and the jacobian can be computed."""
+        wires = 4
+        dev = qml.device('default.qubit.autograd', wires=wires)
+
+        @qml.qnode(dev)

```

Comment:
(just to make it explicit, in case the decorator logic ever changes)
```suggestion
        @qml.qnode(dev, diff_method="backprop")
```